blended-measurement/blended-measurement.tex

```diff
@@ -0,0 +1,685 @@
+    either by directly coadding all images or by coadding images with
+    very small shifts.
+\end{itemize}
+Variable stars and quasars, which are present in every epoch with a different
+flux, can also be treated the same way; as discussed below, it is not clear at
+what stage we should model the variability, but it's reasonable to model them
+at every epoch in at least roughly the same part of the sky.
+
+Transient events that affect only a small fraction of exposures but don't move
+are some what more difficult to handle.  Many of these will be
+straightforwardly detected in single-epoch difference images, and others will
+be detected in special coadds that cover only limited epochs.  However, we
+probably want to mask and reject these objects entirely when building coadds,
+so it will be impossible to deblend them there.  Instead, we'll have to add
+them back in when we transition to multi-epoch measurement (which should be
+straightforward, as their positions will be known, and we'll assume they're
+point sources).
+
+Fast moving solar system objects will similarly be detected in single-epoch
+difference images, and have orbits determined from these detections.  We'll
+also want to mask and reject them from coadds.  We still want to
+include them in multi-epoch measurement, both to ensure overlapping static
+objects are handled correctly and to measure flux as a function of time for
+the moving objects.  We'll use a trailed model for at these (which of course
+approaches a point source as the speed decreases).
+
+Extended variable or transient objects such as comets and supernova light
+echoes will be much harder to model or otherwise deblend in multi-epoch
+measurement, and our assumption for now is that these will be best analyzed
+via difference images, and hence in coaddition and multi-epoch measurement
+we'll simply mask them out.
+
+\section{LSST Pipeline Straw-Man Proposal}
+
+The above sections describe a number of algorithmic options that can be
+combined in myriad ways.  In this section, we describe (at summary level) a
+full baseline pipeline and a few top-priority alternatives. The baseline plan
+is outlined in Figure~\ref{fig:flowchart}, with details described in the next
+section and alternatives in Section~\ref{sec:alternatives}.
+
+\begin{figure}
+\includegraphics[width=\columnwidth]{flowchart}
+\caption{Baseline Pipeline for Blended Measurement}
+\label{fig:flowchart}
+\end{figure}
+
+\subsection{Baseline}
+
+The first major processing stage described here is the Deblender [P1], which
+we imagine as an algorithm very similar to that the SDSS {\em Photo} deblender
+described in Section~\ref{sec:deblender}, likely using a symmetry ansatz to
+define templates.  The inputs will be a detection catalog containing merged
+{\texttt Footprints} and {\texttt Peaks} from all detection images [D1], and
+at least one coadd image per filter [D2].  We may have multiple input coadds
```

Comment:
I think these cases should be rare enough that it will be acceptable to just ignore regions of the sky for which we don't have *any* coadd-quality data.  I also think it's quite likely that we'll define "coadd-quality" in a relative and/or local sense, so if we have one visit in a region it'll always be considered better than nothing.

A trickier problem is when we only have coadd-quality data in some filters, but not all.  We'll need to make sure our algorithms still converge when some filters are missing or extremely low quality, and our masking and selection function characterization will have to indicate this somehow.